src/cli/main.py

```diff
@@ -57,11 +58,7 @@ def execute_command(cmd: str):
 
 
 def main():
-    # TODO: Issue #20
-    project_home = os.path.dirname(os.path.abspath(__file__)) + "/.."
-    os.chdir(project_home)
-
-    _setup_logging()
+    _setup_logging(file=False)
```

Comment:
Logging to file should be on by default or atleast be configurable (and the config option should be turned on by default). This would make it easier to report crash reports for users who only use the gui option (and can not see the stracktrace in the commandline).